packages/app/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM nginxinc/nginx-unprivileged:alpine
```

Comment:
Hm.  Trying to understand which one is the right base image.  I would have assumed it's this: https://hub.docker.com/_/nginx

Reply:
Yeah the problem is that image requires to run as root which is flagged by Sonar and probably other security tools

Reply:
I originally tried with that image. This is the official unprivileged nginx Docker image